code/modules/antagonists/spy_thief/spy_thief.dm

```diff
@@ -7,6 +7,15 @@
 	/// A list of items that this traitor has redeemed using their uplink. This tracks items redeemed with any uplink, so if a spy thief steals another spy thief's uplink, redeemed items will show up here too!
 	var/list/obj/redeemed_items = list()
 
+	New()
+		if (!ticker?.mode?.spy_market)
+			ticker.mode.spy_market = new /datum/game_mode/spy_theft
+			SPAWN(5 SECONDS)
```

Comment:
? missed?

Reply:
Unless I am misunderstanding, if either `ticker` or `mode` are null, the if statement will catch it; I don't see the necessity in checking twice.